### DISCUSSION TITLE
I can't seem to understand how to import a polyfill

### DISCUSSION THREAD

**Discussion by mikealche** [vercel/next.js#11275] [Help] [answered]
Hello! 
First, thanks for this amazing project! It looks really awesome and I'm loving it!! 

However, I'm having lots of troubles setting up [adonisjs' websocket client library](https://adonisjs.com/docs/4.1/websocket-client)

In the documentation says
```
The module build requires the regenerator-runtime polyfill (add it via Babel).

```

But how do I do that? 

Right now I'm trying in .babelrc
```
{
    "presets": [
      [
        "next/babel",
        {
          "preset-env": {},
          "transform-runtime": {},
          "styled-jsx": {},
          "class-properties": {}
        }
      ]
    ],
      "plugins": ["@babel/plugin-transform-regenerator"]
}
```

But that throws me an error saying: 
`Cannot find module '/Users/mike/batcave/workolam/frontend/.next/server/static/development/pages/_document'`

So, how should I import it? 

Question number 2: once I resolve that... how do I import the library so it only loads in the client? 

I would really like to be able to use nextjs as a frontend with adonisjs as a backend. But this configuration error is tearing me down :( 

If anyone could indicate me what steps I should follow, that'd be amazing! 
Thanks in advance

**Comment by timneutkens** [comment, marked as answer]
regenerator-runtime is already included in Next.js, so there's nothing to add really.

**Reply by mikealche** [reply]
Thank you very much Tim! For this, and for all the work you do!
